book/src/getting-started/5-cmake-integration.md

```diff
@@ -10,6 +10,9 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 In this example, we will demonstrate how to integrate cxx-qt code into a C++ application. Cargo builds the cxx-qt code
 as a static library, then CMake links it into a C++ executable.
 
+> If you don't want to use CMake, and only want to use Cargo to build your project, you can [skip to the next chapter](./6-cargo-executable.md).
+> Note that using CMake is currently still easier!
```

Comment:
maybe add to the end `and allows for integrating into existing tooling or projects` ?

Reply:
Done